Pyramid.java

```diff
@@ -1,32 +1,44 @@
-/*
- * File: Pyramid.java
- * Name: 
- * Section Leader: 
- * ------------------
- * This file is the starter file for the Pyramid problem.
- * It includes definitions of the constants that match the
- * sample run in the assignment, but you should make sure
- * that changing these values causes the generated display
- * to change accordingly.
- */
-
-import acm.graphics.*;
-import acm.program.*;
-import java.awt.*;
-
-public class Pyramid extends GraphicsProgram {
-
-/** Width of each brick in pixels */
-	private static final int BRICK_WIDTH = 30;
-
-/** Width of each brick in pixels */
-	private static final int BRICK_HEIGHT = 12;
-
-/** Number of bricks in the base of the pyramid */
-	private static final int BRICKS_IN_BASE = 14;
-	
-	public void run() {
-		/* You fill this in. */
-	}
-}
-
+import acm.graphics.*;

```

Comment:
Great work on this one. Now this is a scalable solution (unlike your solution in ProgramHierarchy.java). Now if we decide to change BRICKS_IN_BASE to 20 (instead of 14) and BRICK_WIDTH to 35 (instead of 30) . Your program still draws the right pyramid without you having to go and change things within the code.